tap_zoom/__init__.py

```diff
@@ -13,8 +13,7 @@
 
 LOGGER = singer.get_logger()
 
-REQUIRED_CONFIG_KEYS = [
-]
+REQUIRED_CONFIG_KEYS = ['client_id', 'client_secret', 'refresh_token']
```

Comment:
Shouldn't this be part of this [PR](https://github.com/singer-io/tap-zoom/pull/20)

Reply:
Removed from here and added the change in https://github.com/singer-io/tap-zoom/pull/20